### PR TITLE
Add support for OSS::AvatarGroup component

### PR DESCRIPTION
**Pull request by phndiaye**
### What does this PR do?

- Add support for OSS::AvatarGroup component
- Add support for loading/placeholder states on OSS::Avatar

### What are the observable changes?

Screenshot 2024-03-29 at 14 54 51
Screenshot 2024-03-29 at 14 54 38

### Good PR checklist
- [x] Title makes sense
- [x] Is against the correct branch
- [x] Only addresses one issue
- [x] Properly assigned
- [x] Added/updated tests
- [x] Added/updated documentation
- [ ] Migrated touched components to Glimmer Components
- [x] Properly labeled

### CODE REVIEW

File: addon/components/o-s-s/avatar.ts [hunk 1]
```diff
@@ -16,6 +16,8 @@ interface OSSAvatarArgs {
   image?: string;
   initials?: string;
   size?: SizeType;
+  loading?: boolean;
+  isPlaceholder?: boolean;
```
Comment (JulienVannier66) [inline addon/components/o-s-s/avatar.ts:20]: Is it really a placeholder here? When it is true, you add a class with a background color

File: app/styles/molecules/avatar-group.less [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+  & > .upf-avatar {
+    background-color: var(--color-white);
+    border: 1px solid var(--color-white);
+    border-radius: 50%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    margin-right: -6px;
```
Comment (JulienVannier66) [inline app/styles/molecules/avatar-group.less:12]: css variable?

File: addon/components/o-s-s/avatar.ts (hunk 1 above)
Comment (phndiaye) [reply addon/components/o-s-s/avatar.ts:20]: don't have any good name 😅  ahah

File: addon/components/o-s-s/avatar-group.stories.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        type: {
+          summary: 'number'
+        },
+        defaultValue: { summary: 'undefined' }
+      },
+      control: {
+        type: 'number'
+      }
+    }
+  }
```
Comment (Miexil) [inline addon/components/o-s-s/avatar-group.stories.js:64]: maybe add a param object with description as we have for other components :
```
parameters: {
    docs: {
      description: {
        component:
          'A component to automaticaly wrap link into an anchor element or a LinkTo helper based on application router knowledge.'
      },
      iframeHeight: 100
    }
  }
```

**Review by Miexil — APPROVED** [review]

**Pull request merged**